app/application.py

```diff
@@ -41,7 +40,7 @@
 else:
     logger.info(" *** APP.Info Testing server settings are being used. ***")
     app.config.from_object(TestingConfig)
-    print ("testing server started...")
+    print("testing server started...")
```

Comment:
Can we make this a log call now? Maybe logging.info("testing serve started...")
:-)